R/antigen.garnish_predict.R

```diff
@@ -269,8 +269,41 @@ parallel::mclapply(dt[, spc %>% unique], function(s){
 
     blastdt <- blastdt[highest == SW] %>%
               .[!(nmer %like% "-")] %>%
-                .[, highest := NULL] %>%
-                  .[nchar(WT.peptide) > 7 & nchar(nmer) > 7]
+                .[, highest := NULL]
+
+    # account for IEDB matches but none long enough to pass to prediction
+    if (nrow(blastdt[nchar(WT.peptide) > 7 & nchar(nmer) > 7]) == 0){
+      message(paste("Only short (<8AA) IEDB matches found, returning iedb_score without passing to affinity prediction."))
```

Comment:
This error message is pretty confusing, I would just elide here.

Reply:
elide???

Can elaborate.  iedb_score calculated but binding affinity of best IEDB match will not be predicted

Reply:
I would just remove this error message.

Reply:
Done